data/test_markers/config_invalid.yml

```diff
@@ -0,0 +1,7 @@
+markers:
+- marker: incorrectly_indented_yaml
+  conditions:
```

Comment:
The label used to be plural but I changed it to singular. This file is meant to have an invalid yaml syntax anyway, but I'll change it to singular since I'm using it to catching a different error.